src/main/java/util/CourseInfoDisplay.java

```diff
@@ -0,0 +1,103 @@
+                            + ")"
+                            + ConsoleColours.RESET
+                            + ":");
+            printDisplay(currentCourse);
+            System.out.println(
+                    " • Press 'Q' to" + ConsoleColours.RED + " quit." + ConsoleColours.RESET);
+            System.out.println(
+                    " • Press '>' to view the"
+                            + ConsoleColours.BLUE
+                            + " next course."
+                            + ConsoleColours.RESET);
+            System.out.println(
+                    " • Press '<' to view the"
+                            + ConsoleColours.BLUE
+                            + " previous course."
+                            + ConsoleColours.RESET);
+
+            char userInput = input.next().charAt(0);
+            switch (userInput) {
+                case 'Q':
+                    userActivity = 'Q';
+                    break;
+                case '<':
+                    if (courseNumber > 0) {
+                        courseNumber--;
+                    } else {
+                        System.out.print(ConsoleColours.RED);
+                        System.out.println("No course before this one.");
+                        System.out.print(ConsoleColours.RESET);
+                    }
+                    break;
+                case '>':
+                    if (courseNumber < numberOfCourses) {
+                        courseNumber++;
+                    } else {
+                        System.out.print(ConsoleColours.RED);
+                        System.out.println("No course after this one.");
+                        System.out.print(ConsoleColours.RESET);
+                    }
+                    break;
+            }
+        }
+    }
+
+    private void printDisplay(Course course) {
+        String description = course.getCourseDesc();
+        System.out.println(
+                ConsoleColours.BLUE_BOLD_BRIGHT
+                        + "-+- "
+                        + course.getCourseId()
+                        + " -+-"
+                        + ConsoleColours.RESET);
+        System.out.println(
+                "---------------------------------------------------------------------------------------------");
+        System.out.println(
+                ConsoleColours.WHITE_BOLD_BRIGHT + "Course Description:" + ConsoleColours.RESET);
+        for (String s : description.split("(?<=\\.) (?=[A-Z])")) {
+            System.out.println(s);
+        }
+        System.out.println(
+                "---------------------------------------------------------------------------------------------");
+        System.out.println(
+                "Exclusions: "
+                        + ConsoleColours.BLUE
+                        + course.getExclusions().toString()
+                        + ConsoleColours.RESET);
+        System.out.println(
+                "Corequisites: "
```

Comment:
When a course has a recommended corequisite, the program prints `Corequisites: [Recomm]`. An example would be MAT332F.